### PR TITLE
Migrate Sites to Pantheon doc - audit

### LINKED ISSUE #1328: Migrate Sites to Pantheon Doc Update
Re: [Migrate Sites to Pantheon](https://pantheon.io/docs/articles/sites/migrate)
## Audit

- [x] Remove unnecessary intro content.
- [x] align headers and content to style guide
- [x] correct typos/grammatical errors 
- [x] change code blocks to code spans when needed

### PR DESCRIPTION
**Pull request by nataliejeremy**
Closes #1328 
@bmackinney please review

### CODE REVIEW

File: source/docs/articles/sites/migrate.md [hunk 1]
```diff
@@ -9,108 +9,98 @@ category:
...
-### Evaluate Your Site
+- **Files** - Anything in `sites/default/files` for Drupal or `wp-content/uploads` for WordPress. This houses a combination of uploaded content from site users, along with generated stylesheets, aggregated scripts, image styles, etc.
 
-1. Review your codebase.
- - Are you running another application in addition to WordPress or Drupal? See [Platform Considerations](/docs/articles/sites/platform-considerations/#one-application-per-site).
-2. Measure your database size.
-3. Locate your non-version-controlled static assets.
-4. Review your logs.
 
 ### Prepare Your Site For Export
```
Comment (bmackinney) [inline source/docs/articles/sites/migrate.md]: This section belongs at the beginning of the parent section

Comment (bmackinney) [inline source/docs/articles/sites/migrate.md]: Actually I take that back..

File: source/docs/articles/sites/migrate.md [hunk 2]
```diff
@@ -9,108 +9,98 @@ category:
...
+* **Clear all caches**. This removes unnecessary and out-of-date files from both the database and your filesystem, which will save time and valuable space.
 * Take a look at your codebase and **remove any non-core code from your site** that you aren’t planning on running on Pantheon.
 * If you’ve been using the database for things other than Drupal or WordPress, you should **drop or skip any unnecessary or unrelated database tables** that your site doesn’t need.
 
 ### Plan the Import
-You can import during the site creation process using the importer tool or manually after the site has been created. Importing after creation requires code to be upload via git, along with separate imports of your database and files. Existing Drupal 8 sites must be imported manually.
+You can import during the site creation process using the importer tool or manually after the site has been created. Importing after creation requires you to upload code via Git, along with separate imports of your database and files. Existing Drupal 8 sites must be imported manually.
```
Comment (bmackinney) [inline source/docs/articles/sites/migrate.md]: I think we can remove

> Importing after creation requires you to upload code via Git, along with separate imports of your database and files. Existing Drupal 8 sites must be imported manually.

File: source/docs/articles/sites/migrate.md [hunk 3]
```diff
@@ -9,108 +9,98 @@ category:
...
 
 <div class="alert alert-danger" role="alert"><h4>Reminder</h4>Importing automatically upgrades to the latest version of core. It's a best practice to keep core up-to-date to benefit from security and bug fixes, but if you use a site or distribution that relies on an outdated version of core, you may experience incompatibilities. If you experience issues, see the troubleshooting documentation for your <a href="https://codex.wordpress.org/Updating_WordPress#Troubleshooting">WordPress</a> or <a href="https://www.drupal.org/troubleshooting"> Drupal</a> upstream.</div>
 
-The importer accepts either single-file site archives or separate archives of the code, database, and files. It accepts file uploads up to 100MB, and can download publicly-accessible archives up to 500MB. Acceptable file types include tar, zip, or gzip.
+The importer accepts either single file site archives or separate archives of the code, database, and files. It accepts file uploads up to 100MB, and can download publicly-accessible archives up to 500MB. Acceptable file types include tar, zip, or gzip.
```
Comment (bmackinney) [inline source/docs/articles/sites/migrate.md]: add `.sql` for databases

File: source/docs/articles/sites/migrate.md [hunk 4]
```diff
@@ -9,108 +9,98 @@ category:
...
 
 ### Create Separate Archives of Code, Database, and Files
 
-If your site cannot be packaged as a single archive less than 500MB, or you need to use an upstream other than "Vanilla" Drupal or WordPress, you'll need to create separate archives of each part of your site. For step-by-step instructions, see [Exporting an Existing WordPress Site](/docs/articles/sites/migrate/export-an-existing-wordpress-site#manually-create-separate-site-archives) or [Exporting an Existing Drupal Site](/docs/articles/sites/migrate/export-an-existing-drupal-site#manually-create-archive).
-
-## Move In
+If your site cannot be packaged as a single archive less than 500MB, or you need to use an upstream other than "vanilla" Drupal or WordPress, you'll need to create separate archives of each part of your site. For step-by-step instructions, see [Exporting an Existing WordPress Site](/docs/articles/sites/migrate/export-an-existing-wordpress-site#manually-create-separate-site-archives) or [Exporting an Existing Drupal Site](/docs/articles/sites/migrate/export-an-existing-drupal-site#manually-create-archive).
```
Comment (bmackinney) [inline source/docs/articles/sites/migrate.md]: remove: 
>or you need to use an upstream other than "vanilla" Drupal or WordPress,

**Comment by rachelwhitton** [comment]
+1 - Great audit, thanks @nataliejeremy!

**Pull request merged**